src/Uno.UI/Behaviors/VisibleBoundsPadding.cs

```diff
@@ -348,11 +348,14 @@ private Thickness CalculateAppliedPadding(PaddingMask mask, Thickness visibility
 			private void ApplyPadding(Thickness padding)
 			{
 #if HAS_UNO // Is building using Uno.UI
-				if (Owner is { } owner
-					&& owner.SetPadding(padding)
-					&& _log.IsEnabled(LogLevel.Debug))
+				if (Owner is { } owner && owner.SetPadding(padding))
 				{
-					_log.LogDebug($"ApplyPadding={padding}");
+#if __ANDROID__
+					// Dispatching on Android prevents issues where layout/render changes occurring
+					// during initial loading of the view are not always properly picked up by the layouting/rendering engine.
+					_ = owner.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, owner.InvalidateMeasure);
+#endif
+					_log.Log(LogLevel.Debug, $"ApplyPadding={padding}");
```

Comment:
dont drop the if-guard:
```suggestion
					if (_log.IsEnabled(LogLevel.Debug))
					{
						_log.Log(LogLevel.Debug, $"ApplyPadding={padding}");
					}

#if __ANDROID__
					// Dispatching on Android prevents issues where layout/render changes occurring
					// during initial loading of the view are not always properly picked up by the layouting/rendering engine.
					_ = owner.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, owner.InvalidateMeasure);
```

Reply:
The `.Log(LogLevel, ...)` method already includes the guard, no?

https://github.com/unoplatform/uno/blob/57248d0a6712a8f3796ad680ad4fa9f9f23b87f4/src/Uno.Foundation.Logging/Logger.cs#L21-L27

Reply:
oh nevermind i see the issue